internal/interfaces/grpc/service.go

```diff
@@ -339,8 +341,22 @@ func (s *service) start(withUnlockerOnly bool) (*services, error) {
 		daemonv1.RegisterWalletServiceServer(grpcOperatorServer, walletHandler)
 	}
 
+	tdexConnectSvc := httpinterface.NewTdexConnectService(
+		s.opts.WalletUnlockerSvc,
+		adminMacaroonPath,
+		s.opts.operatorTLSCert(),
+		operatorAddr,
+	)
+
 	// http Operator server for grpc-web
-	http1OperatorServer, http2OperatorServer := newGRPCWrappedServer(operatorAddr, grpcOperatorServer, nil)
+	http1OperatorServer, http2OperatorServer := newGRPCWrappedServer(
+		operatorAddr,
+		grpcOperatorServer,
+		nil,
+		map[string]func(w http.ResponseWriter, req *http.Request){
+			"/tdexconnect": tdexConnectSvc.TdexConnectHandler,
```

Comment:
Maybe best to serve from root index, ideally user just click on deployed instance IP/Domain and should be able to show this
```suggestion
			"/": tdexConnectSvc.TdexConnectHandler,
```

Reply:
Done